src/ListGroup.js

```diff
@@ -9,6 +9,45 @@ class ListGroup extends React.Component {
       (item, index) => cloneElement(item, { key: item.key ? item.key : index })
     );
 
+    let childrenAnchors = false;
+
+    if (!this.props.children) {
+      return this.renderDiv(items);
+    } else {
+
+      if (Array.isArray(this.props.children)) {
+        this.props.children.forEach((child) => {
+          if (child.props.href) {
+            childrenAnchors = true;
+          }
+        });
```

Comment:
As for `Array.isArray(this.props.children)`

The `this.props.children` has opaque type and structure.
Consequently it can be subject to changes in future React releases.
Perhaps it would be more appropriate to use here [React.Children](https://facebook.github.io/react/docs/top-level-api.html#react.children) API ?